README.md

```diff
@@ -1,51 +1,56 @@
-BCE App
-=======
+# BCE App
+
 This repository contains my solution to the problem of creating variable risk types for the BriteCore Engineering Code Challenge.
 The app itself is live and can be accessed from the urls in the tl;dr
 
 # TL;DR
+
 ## Deliverables
 
 1. This file
 2. Links to the live app are [Listed Here](#ui)
-3. Model file can be [found here](bce-server/bce/models.py) and [here](doc/models.py)
-4. The entity diagram can be [found here](doc/Entities.png), and  [here](#entities)
-5. Backend Api link can is [shown here](#server) 
+3. Model file can be [found here](server/bce/models.py)
+4. The entity diagram can be [found here](doc/Entities.png), and [here](#entities)
+5. Backend Api link can is [shown here](#server)
 6. Can be checked [here](bce-server/bce/views.py)
-7. Live UI app is linked [here](#ui) along with the details to log in and test 
+7. Live UI app is linked [here](#ui) along with the details to log in and test
```

Comment:
what changed here or did you forget to add the change?

Reply:
A trailing space was removed.